pilates/urbansim/preprocessor.py

```diff
@@ -96,12 +95,10 @@ def add_skims_to_model_data(settings, data_dir=None):
     # load datastore
     region_id = settings['region_to_region_id'][region]
     model_data_fname = usim_model_data_fname(region_id)
-    if not data_dir:
-        data_dir = settings['usim_local_data_folder']
-    model_data_fpath = os.path.join(data_dir, model_data_fname)
-    if not os.path.exists(model_data_fpath):
-        raise ValueError('No input data found at {0}'.format(
-            model_data_fpath))
+    data_dir = settings['data_folder'] / settings['usim_local_data_folder']
```

Comment:
leave `data_dir` as an optional setting

Reply:
I'm pretty sure I went through the codebase and it (`data_dir`) is never passed as an argument to this function. So given that the default was always used... I just removed it as an argument and gave it the same default value it was getting everytime anyway... does that make sense?

Reply:
This should be ok. I thought we wanted the function to save certain outputs in a `tmp` folder, but it is not needed.